src/components/patients/patientTable/index.tsx

```diff
@@ -2,8 +2,8 @@ import React from "react";
 import { Link } from "react-router-dom";
 import { Cell, Column, Table } from "@blueprintjs/table";
 import { Icon, H3 } from "@blueprintjs/core";
-import { DATA_TEST } from "constants/dataTest";
 import { ROUTE_PATIENT } from "constants/routes";
+import { getPatients } from "services/ApiCalls";
```

Comment:
file names should never start with a capital letter. rename to `services/api` instead

Reply:
Done